### ISSUE TITLE
Build failing for scikit-surgery

### ISSUE THREAD

**Issue by mianasbat** [UCL/scikit-surgery#67]
It seems that the build is failing for master branch of project scikit-surgery (https://github.com/UCL/scikit-surgery) . Is it because of the migration issues?

@tdowrick @thompson318 @MattClarkson

**Comment by mianasbat** [comment]
The test log shows error `No file or directory present` on line 44 in `test_docs_complete.py`. I was exploring it and I found that os.listdir() is given relative path. I gave it absolute path i.e. `/Users/mianasbatahmad/temp/scikit-surgery/modules` and that specific error disappeared. I also found that os.listdir() is not very comfortable with relative path as discussed on https://bit.ly/2ZP838u
I am checking it more. Open for any hints, suggestions!

**Comment by mianasbat** [comment]
Tom and I looked it this morning and that was very helpful. So one reason the tests were failing on my computer was that my modules directory contained .DS_Store directory and the code was looking for docs in all directories inside module directory. We added code to only consider directories starting with scikit- to fix it.
 
Secondly the submodules are empty directories (linked to respective repose) because I used `git clone scikit-surgery` when I should have used `git clone --recursive scikit-surgery` to download the submodules too as pointed by Tom. After this the file not found error should go away (ideally)

We will need to figure out how to manage it in online testing.
